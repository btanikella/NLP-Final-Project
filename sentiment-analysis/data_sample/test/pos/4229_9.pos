PRP VBD VBG DT TO VB RB IN DT NNS  VBZ TO VB JJ : VBZ RP VBG RB
RB IN DT NN DT NNS MD VB VBN DT CC RB VBD DT NN IN DT RB JJ NN RB VBG DT JJ NNS
DT VBZ IN DT JJ NN RB IN DT JJ NN IN DT NNP POS NNP NN
DT JJ NN VBZ VBG DT DT JJS IN DT NN IN RB
