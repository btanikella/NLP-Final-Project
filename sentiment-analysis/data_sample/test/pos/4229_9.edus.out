Sentence #1 (21 tokens):
I was expecting this to be just like the others, tries to be scary- ends up looking silly.
[Text=I CharacterOffsetBegin=4 CharacterOffsetEnd=5 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=6 CharacterOffsetEnd=9 PartOfSpeech=VBD] [Text=expecting CharacterOffsetBegin=10 CharacterOffsetEnd=19 PartOfSpeech=VBG] [Text=this CharacterOffsetBegin=20 CharacterOffsetEnd=24 PartOfSpeech=DT] [Text=to CharacterOffsetBegin=25 CharacterOffsetEnd=27 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=28 CharacterOffsetEnd=30 PartOfSpeech=VB] [Text=just CharacterOffsetBegin=31 CharacterOffsetEnd=35 PartOfSpeech=RB] [Text=like CharacterOffsetBegin=36 CharacterOffsetEnd=40 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=41 CharacterOffsetEnd=44 PartOfSpeech=DT] [Text=others CharacterOffsetBegin=45 CharacterOffsetEnd=51 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=51 CharacterOffsetEnd=52 PartOfSpeech=,] [Text=tries CharacterOffsetBegin=53 CharacterOffsetEnd=58 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=59 CharacterOffsetEnd=61 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=62 CharacterOffsetEnd=64 PartOfSpeech=VB] [Text=scary CharacterOffsetBegin=65 CharacterOffsetEnd=70 PartOfSpeech=JJ] [Text=- CharacterOffsetBegin=70 CharacterOffsetEnd=71 PartOfSpeech=:] [Text=ends CharacterOffsetBegin=72 CharacterOffsetEnd=76 PartOfSpeech=VBZ] [Text=up CharacterOffsetBegin=77 CharacterOffsetEnd=79 PartOfSpeech=RP] [Text=looking CharacterOffsetBegin=80 CharacterOffsetEnd=87 PartOfSpeech=VBG] [Text=silly CharacterOffsetBegin=88 CharacterOffsetEnd=93 PartOfSpeech=RB] [Text=. CharacterOffsetBegin=93 CharacterOffsetEnd=94 PartOfSpeech=.] 
root(ROOT-0, expecting-3)
nsubj(expecting-3, I-1)
aux(expecting-3, was-2)
nsubj(tries-12, this-4)
aux(like-8, to-5)
aux(like-8, be-6)
advmod(like-8, just-7)
prep(tries-12, like-8)
det(others-10, the-9)
pobj(like-8, others-10)
punct(tries-12, ,-11)
csubj(ends-17, tries-12)
aux(scary-15, to-13)
cop(scary-15, be-14)
dep(ends-17, scary-15)
punct(ends-17, --16)
ccomp(expecting-3, ends-17)
prt(ends-17, up-18)
xcomp(ends-17, looking-19)
advmod(looking-19, silly-20)
punct(expecting-3, .-21)

Sentence #2 (26 tokens):
Somewhere along the line the writers must have realised this and so took the film in a totally different direction basically ignoring the other films.
[Text=Somewhere CharacterOffsetBegin=99 CharacterOffsetEnd=108 PartOfSpeech=RB] [Text=along CharacterOffsetBegin=109 CharacterOffsetEnd=114 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=115 CharacterOffsetEnd=118 PartOfSpeech=DT] [Text=line CharacterOffsetBegin=119 CharacterOffsetEnd=123 PartOfSpeech=NN] [Text=the CharacterOffsetBegin=124 CharacterOffsetEnd=127 PartOfSpeech=DT] [Text=writers CharacterOffsetBegin=128 CharacterOffsetEnd=135 PartOfSpeech=NNS] [Text=must CharacterOffsetBegin=136 CharacterOffsetEnd=140 PartOfSpeech=MD] [Text=have CharacterOffsetBegin=141 CharacterOffsetEnd=145 PartOfSpeech=VB] [Text=realised CharacterOffsetBegin=146 CharacterOffsetEnd=154 PartOfSpeech=VBN] [Text=this CharacterOffsetBegin=155 CharacterOffsetEnd=159 PartOfSpeech=DT] [Text=and CharacterOffsetBegin=160 CharacterOffsetEnd=163 PartOfSpeech=CC] [Text=so CharacterOffsetBegin=164 CharacterOffsetEnd=166 PartOfSpeech=RB] [Text=took CharacterOffsetBegin=167 CharacterOffsetEnd=171 PartOfSpeech=VBD] [Text=the CharacterOffsetBegin=172 CharacterOffsetEnd=175 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=176 CharacterOffsetEnd=180 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=181 CharacterOffsetEnd=183 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=184 CharacterOffsetEnd=185 PartOfSpeech=DT] [Text=totally CharacterOffsetBegin=186 CharacterOffsetEnd=193 PartOfSpeech=RB] [Text=different CharacterOffsetBegin=194 CharacterOffsetEnd=203 PartOfSpeech=JJ] [Text=direction CharacterOffsetBegin=204 CharacterOffsetEnd=213 PartOfSpeech=NN] [Text=basically CharacterOffsetBegin=214 CharacterOffsetEnd=223 PartOfSpeech=RB] [Text=ignoring CharacterOffsetBegin=224 CharacterOffsetEnd=232 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=233 CharacterOffsetEnd=236 PartOfSpeech=DT] [Text=other CharacterOffsetBegin=237 CharacterOffsetEnd=242 PartOfSpeech=JJ] [Text=films CharacterOffsetBegin=243 CharacterOffsetEnd=248 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=248 CharacterOffsetEnd=249 PartOfSpeech=.] 
root(ROOT-0, along-2)
advmod(along-2, Somewhere-1)
det(line-4, the-3)
pobj(along-2, line-4)
det(writers-6, the-5)
nsubj(realised-9, writers-6)
aux(realised-9, must-7)
aux(realised-9, have-8)
rcmod(line-4, realised-9)
dobj(realised-9, this-10)
cc(realised-9, and-11)
advmod(took-13, so-12)
conj(realised-9, took-13)
det(film-15, the-14)
dobj(took-13, film-15)
prep(took-13, in-16)
det(direction-20, a-17)
advmod(different-19, totally-18)
amod(direction-20, different-19)
pobj(in-16, direction-20)
advmod(ignoring-22, basically-21)
partmod(took-13, ignoring-22)
det(films-25, the-23)
amod(films-25, other-24)
dobj(ignoring-22, films-25)
punct(along-2, .-26)

Sentence #3 (18 tokens):
This feels like a different film rather than the fourth entry in the Child's Play series.
[Text=This CharacterOffsetBegin=254 CharacterOffsetEnd=258 PartOfSpeech=DT] [Text=feels CharacterOffsetBegin=259 CharacterOffsetEnd=264 PartOfSpeech=VBZ] [Text=like CharacterOffsetBegin=265 CharacterOffsetEnd=269 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=270 CharacterOffsetEnd=271 PartOfSpeech=DT] [Text=different CharacterOffsetBegin=272 CharacterOffsetEnd=281 PartOfSpeech=JJ] [Text=film CharacterOffsetBegin=282 CharacterOffsetEnd=286 PartOfSpeech=NN] [Text=rather CharacterOffsetBegin=287 CharacterOffsetEnd=293 PartOfSpeech=RB] [Text=than CharacterOffsetBegin=294 CharacterOffsetEnd=298 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=299 CharacterOffsetEnd=302 PartOfSpeech=DT] [Text=fourth CharacterOffsetBegin=303 CharacterOffsetEnd=309 PartOfSpeech=JJ] [Text=entry CharacterOffsetBegin=310 CharacterOffsetEnd=315 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=316 CharacterOffsetEnd=318 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=319 CharacterOffsetEnd=322 PartOfSpeech=DT] [Text=Child CharacterOffsetBegin=323 CharacterOffsetEnd=328 PartOfSpeech=NNP] [Text='s CharacterOffsetBegin=328 CharacterOffsetEnd=330 PartOfSpeech=POS] [Text=Play CharacterOffsetBegin=331 CharacterOffsetEnd=335 PartOfSpeech=NNP] [Text=series CharacterOffsetBegin=336 CharacterOffsetEnd=342 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=342 CharacterOffsetEnd=343 PartOfSpeech=.] 
root(ROOT-0, feels-2)
nsubj(feels-2, This-1)
prep(feels-2, like-3)
det(film-6, a-4)
amod(film-6, different-5)
pobj(like-3, film-6)
advmod(than-8, rather-7)
prep(film-6, than-8)
det(entry-11, the-9)
amod(entry-11, fourth-10)
pobj(than-8, entry-11)
prep(entry-11, in-12)
det(Child-14, the-13)
poss(series-17, Child-14)
possessive(Child-14, 's-15)
nn(series-17, Play-16)
pobj(in-12, series-17)
punct(feels-2, .-18)

Sentence #4 (14 tokens):
The new idea works making this the best in the series by far.
[Text=The CharacterOffsetBegin=348 CharacterOffsetEnd=351 PartOfSpeech=DT] [Text=new CharacterOffsetBegin=352 CharacterOffsetEnd=355 PartOfSpeech=JJ] [Text=idea CharacterOffsetBegin=356 CharacterOffsetEnd=360 PartOfSpeech=NN] [Text=works CharacterOffsetBegin=361 CharacterOffsetEnd=366 PartOfSpeech=VBZ] [Text=making CharacterOffsetBegin=367 CharacterOffsetEnd=373 PartOfSpeech=VBG] [Text=this CharacterOffsetBegin=374 CharacterOffsetEnd=378 PartOfSpeech=DT] [Text=the CharacterOffsetBegin=379 CharacterOffsetEnd=382 PartOfSpeech=DT] [Text=best CharacterOffsetBegin=383 CharacterOffsetEnd=387 PartOfSpeech=JJS] [Text=in CharacterOffsetBegin=388 CharacterOffsetEnd=390 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=391 CharacterOffsetEnd=394 PartOfSpeech=DT] [Text=series CharacterOffsetBegin=395 CharacterOffsetEnd=401 PartOfSpeech=NN] [Text=by CharacterOffsetBegin=402 CharacterOffsetEnd=404 PartOfSpeech=IN] [Text=far CharacterOffsetBegin=405 CharacterOffsetEnd=408 PartOfSpeech=RB] [Text=. CharacterOffsetBegin=408 CharacterOffsetEnd=409 PartOfSpeech=.] 
root(ROOT-0, works-4)
det(idea-3, The-1)
amod(idea-3, new-2)
nsubj(works-4, idea-3)
xcomp(works-4, making-5)
dobj(making-5, this-6)
det(best-8, the-7)
dep(this-6, best-8)
prep(best-8, in-9)
det(series-11, the-10)
pobj(in-9, series-11)
prep(best-8, by-12)
pobj(by-12, far-13)
punct(works-4, .-14)

